clients/admin-ui/package-lock.json

```diff
@@ -19427,11 +19252,6 @@
       "integrity": "sha512-PkUpF+qoXTqhOeWL9fu7As8LXsIUZ1WYaJiY/a7McAQzxjk82OF0tibkFXVCDImZtWxbvojFjerkiLb0/q8mew==",
       "dev": true
     },
-    "pretty-format": {
-      "version": "3.8.0",
-      "resolved": "https://registry.npmjs.org/pretty-format/-/pretty-format-3.8.0.tgz",
-      "integrity": "sha1-v77VbV6ad2ZF9LH/eqGjrE+jw4U="
-    },
```

Comment:
Just making sure all these changes to the package-lock file were intentionally committed.

Reply:
hmmm this may have been generated during a build? I'm unsure if I should commit - cc @TheAndrewJackson

Reply:
I'm not 100% sure either. I double checked and it's not a dependency in our `package.json`. My guess is that it one our dependencies, let's say `prettier`, used to use this package and recently removed it. When you did an `npm install` it may have updated the `package-lock.json` and removed it because our dependency removed it. Since the tests are passing it seems safe.

Does everything still run fine on your local machine?

Reply:
that makes sense Andrew. everything still runs fine, I was just making sure that that's what we're doing, we just commit the latest `package-lock.json` that was generated.

Reply:
yup everything runs just fine locally